orttraining/tools/ci_test/download_e2e_test_data.py

```diff
@@ -47,15 +43,18 @@ def _check_file_sha256_digest(path, expected_digest):
 
 def main():
   parser = argparse.ArgumentParser(description="Downloads training end-to-end test data.")
-  parser.add_argument("target_dir", help="The test data destination directory.")
+  parser.add_argument("--azure_blob_url", required=True, help="The test data destination directory.")
```

Comment:
this help text is not accurate. same comment for "archive_sha256_digest". also why do we need to make the URL and digest command line arguments? it makes manually calling this script a bit more difficult. they're still effectively hardcoded in the build definition.